src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -3009,6 +3009,12 @@ To install the latest version of a package, click Install. \n
   <data name="WorkspaceTabTooltipHeaderUnsaved" xml:space="preserve">
     <value>Unsaved</value>
   </data>
+  <data name="TourLabelProgressText" xml:space="preserve">
+    <value>of</value>
+  </data>
+  <data name="PackagesGuideSearchResultsTitle" xml:space="preserve">
```

Comment:
This is missing in `en-US`

Reply:
@QilongTang that is the bug, that the string is already in en-US but missing in this file, that why is not translated to other languages.
https://github.com/DynamoDS/Dynamo/blob/3e22a571a202ce9c38aa56a6f9b64dd45c116c02/src/DynamoCoreWpf/Properties/Resources.en-US.resx#L2977